src/plugins/editor/components/editor.jsx

```diff
@@ -161,21 +166,22 @@ export default function makeEditor({ editorPluginsToRun }) {
       })
     }
 
-    updateYaml = () => {
-      if(this.editor.getValue() === this.yaml[0]) {
```

Comment:
Any reason this check was dropped? I don't imagine it could hurt anything.

Reply:
The changes in this PR are all about improving this area. Ie: only update editor when the `origin` is different. We can include this check if you like, but this check is not the core logic behind our defence against bad re-renders